Framework/Subset_of_CorLib/System/Single.cs

```diff
@@ -22,14 +22,34 @@ public struct Single
 
         public override String ToString()
         {
-            return Number.Format(m_value, false, "G", NumberFormatInfo.CurrentInfo);
+            // Number.Format method is responsible for returning the correct string representation of the value; however, it does not work properly for special values.

```

Comment:
Can you reflow the comments here, they are really long and don't fit on the screen in a GitHub review...